Embedded struct can not be used with -transform camelcase

```golang
type PDetail struct {
	PSummary //will cause problem
	Thumbnail             string `json:"thumbnail"`
}
```

```
 W
panic: runtime error: index out of range

goroutine 1 [running]:
main.(*config).addTags(0xc4200a4200, 0x0, 0x0, 0xc4200e74e0, 0x0, 0x100eda8, 0x20)
        /Users/ / /go/src/github.com/fatih/gomodifytags/main.go:351 +0x8a5
```

--- a/main.go
+++ b/main.go
@@ -585,8 +585,18 @@ func (c *config) rewriteFields(node ast.Node) (ast.Node, error) {
 			}
 
 			fieldName := ""
-			if f.Names != nil && len(f.Names) != 0 {
+			if len(f.Names) != 0 {
 				fieldName = f.Names[0].Name
+			}
+
+			// anonymous field
+			if f.Names == nil {
+				ident, ok := f.Type.(*ast.Ident)
+				if !ok {
+					continue
+				}
+
+				fieldName = ident.Name
 			}
 
 			res, err := c.process(fieldName, f.Tag.Value)

--- a/main_test.go
+++ b/main_test.go
@@ -204,6 +204,15 @@ func TestRewrite(t *testing.T) {
 			},
 		},
 		{
+			file: "line_camelcase_add_embedded",
+			cfg: &config{
+				add:       []string{"json"},
+				output:    "source",
+				line:      "4,6",
+				transform: "camelcase",
+			},
+		},
+		{
 			file: "line_value_add",
 			cfg: &config{
 				add:    []string{"json:foo"},

--- a/test-fixtures/line_camelcase_add_embedded.golden
+++ b/test-fixtures/line_camelcase_add_embedded.golden
@@ -1,0 +1,7 @@
+package foo
+
+type foo struct {
+	bar       `json:"bar"`
+	MyExample bool     `json:"myExample"`
+	MyAnother []string `json:"myAnother"`
+}

--- a/test-fixtures/line_camelcase_add_embedded.input
+++ b/test-fixtures/line_camelcase_add_embedded.input
@@ -1,0 +1,7 @@
+package foo
+
+type foo struct {
+	bar
+	MyExample bool
+	MyAnother []string
+}

